### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by keithluchtel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  bear: {},
+});
+
+const persistStore = persist([
+  [rootStore.bear, {key: "bear", storage: AsyncStorage}],
+  [rootStore.fish, {key: "fish", storage: AsyncStorage}],
+])
+```
+
+### Rehydraed?
```
Comment (carlos-kelly) [inline README.md:40]: Rehydrated?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    // app content
+  );
+});
+```
+
+## Options
+
+| option | Description |
+| ------ | ----------- |
+| key | **required** - key data is stored under within the Storage Engine. |
```
Comment (carlos-kelly) [inline README.md:66]: Under within?

**Review by carlos-kelly — COMMENTED** [review]
A couple comments overall pretty good.

File: README.md (hunk 2 above)
Comment (keithluchtel) [reply README.md]: updated

File: README.md (hunk 1 above)
Comment (keithluchtel) [reply README.md]: fixed

**Review by carlos-kelly — APPROVED** [review]

**Review by gksander — APPROVED** [review]
Neat. Might want to sneak a changeset in so you can release this up to NPM

**Pull request merged**